packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+| ------------ | ---------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `appearance` | appearance |                                                                                                                                  |
+| `disabled`   | disabled   |                                                                                                                                  |
+| `multiple`   | multiple   | [FASTListbox](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/listbox/listbox.spec.md) |
+| `open`       | open       |                                                                                                                                  |
+| `size`       | size       |                                                                                                                                  |
+
+<br />
+
+### **CSS Parts**
+
+| Name             | Description                                          |
+| ---------------- | ---------------------------------------------------- |
+| `control`        | The element representing the select invoking element |
+| `indicator`      | The element wrapping the visual indicator            |
+| `listbox`        | The listbox element                                  |
+| `selected-value` | The element wrapping the selected value              |
+
+### **Slots**
+
+| Name               | Description                                                                                                                         |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------- |
+|                    | the list of options, either <fluent-option> or elements with role="option".                                                         |
+| `button-container` | contains the selected value and indicator slots inside the control element.                                                         |
+| `content`          | the content of the button                                                                                                           |
+| `end`              | often times a glyph, icon, or button follows the content                                                                            |
+| `indicator`        | holds the glyph indicating that the select can be expanded. This slot is only available if the button-container slot is not filled. |
+| `selected-value`   | displays the currently selected value text. This slot is only available if the button-container slot is not filled.                 |
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+<br />
+
+`selectTemplate` from FastFoundation
```

Comment:
I see this called out here but the initial implementation duplicates it - is there a reason for the delta?